src/Database/Behaviors/Sortable.php

```diff
@@ -23,6 +23,7 @@
  *
  *   const SORT_ORDER = 'my_sort_order';
  *
+ * @deprecated 1.2.0. We recommend using the \Winter\Storm\Database\Traits\Sortable trait instead.
```

Comment:
Why? The DB traits as behaviours are intended for use by plugins extending other plugins.

Reply:
Yeah, I added this to dynamically make a model Sortable when extending a plugin.